fs/littlefs/CMakeLists.txt

```diff
@@ -51,17 +51,18 @@ if(CONFIG_FS_LITTLEFS)
 
   target_compile_definitions(
     fs
-    PRIVATE $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_TRACE=finfo>
-            $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_DEBUG=finfo>
-            $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_WARN=fwarn>
-            $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_ERROR=ferr>
-            $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_ASSERT=DEBUGASSERT>
-            $<$<NOT:$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>>:-DLFS_CONFIG=${CMAKE_CURRENT_LIST_DIR}/lfs_vfs.h>
-            $<$<BOOL:${CONFIG_FS_LITTLEFS_HAS_LFS_DEFINES}>:-DLFS_DEFINES=${CMAKE_CURRENT_LIST_DIR}/lfs_nuttx_defines.h>
-            -DLFS_NAME_MAX=${CONFIG_FS_LITTLEFS_NAME_MAX}
-            -DLFS_FILE_MAX=${CONFIG_FS_LITTLEFS_FILE_MAX}
-            -DLFS_ATTR_MAX=${CONFIG_FS_LITTLEFS_ATTR_MAX}
-            $<$<BOOL:${CONFIG_FS_LITTLEFS_MULTI_VERSION}>:-DLFS_MULTIVERSION>)
+    PRIVATE
```

Comment:
why not done in the first place

Reply:
?

Reply:
I mean merging the last and first patch into one.

Reply:
ok
it was a separate commit just because i keep forgetting the fact that the ci performs cmake-format check.

Reply:
done